src/components/Search/index.tsx

```diff
@@ -294,13 +294,15 @@ function Search({queryJSON}: SearchProps) {
             SearchActions.createTransactionThread(hash, item.transactionID, reportID, item.moneyRequestReportActionID);
         }
 
+        const backTo = ROUTES.SEARCH_CENTRAL_PANE.getRoute({query: queryJSON.inputQuery});
+
         if (SearchUtils.isReportActionListItemType(item)) {
             const reportActionID = item.reportActionID;
-            Navigation.navigate(ROUTES.SEARCH_REPORT.getRoute(reportID, reportActionID));
+            Navigation.navigate(ROUTES.SEARCH_REPORT.getRoute({reportID, reportActionID, backTo}));
```

Comment:
Could we use `Navigation.getActiveRoute()` instead of passing the `queryJSON.inputQuery` and building the route? Same for the other instances of backTo in this PR

Reply:
This is a great improvement Carlos! thanks, I tested and it works.

The only small difference is that if I generate `backTo` it looks like this: `search?foo` whereas generated from `getActiveRoute()` it has the extra leading slash: `/search?foo`.
But as far as I can see it makes no changes in navigation logic, so I'm fixing this 👍

Reply:
Nice! Yea, I think that should be ok. Thanks for the changes!